renderer/src/shared/TimeBadge/__tests__/TimeBadge.test.tsx

```diff
@@ -0,0 +1,140 @@
+import TimeBadge from "../TimeBadge";
+import { TimeBadgeProps } from "../types";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+
+const day = 60 * 60 * 24 * 1000;
+const today = new Date();
+const yesterday = new Date(today.getTime() - day);
+
+const yesterdayLess6MockedProps: TimeBadgeProps = {
+  hours: 5,
+  startTime: "08:00",
+  selectedDate: yesterday,
+};
+const yesterdayLess8MockedProps: TimeBadgeProps = {
+  hours: 7,
+  startTime: "08:00",
+  selectedDate: yesterday,
+};
+
+const todayStartWorkLess6MockedProps: TimeBadgeProps = {
+  hours: 5,
+  startTime: "10:00",
+  selectedDate: today,
+};
+const todayStartWorkLess8MockedProps: TimeBadgeProps = {
+  hours: 7,
+  startTime: "10:00",
+  selectedDate: today,
+};
+
+const todayEndWorkLess6MockedProps: TimeBadgeProps = {
+  hours: 5,
+  startTime: "02:00",
+  selectedDate: today,
+};
+const todayEndWorkLess8MockedProps: TimeBadgeProps = {
+  hours: 7,
+  startTime: "02:00",
+  selectedDate: today,
+};
+const yesterdayMore8MockedProps: TimeBadgeProps = {
+  hours: 9,
+  startTime: "10:00",
+  selectedDate: yesterday,
+};
+const todayMore8MockedProps: TimeBadgeProps = {
+  hours: 9,
+  startTime: "10:00",
+  selectedDate: today,
+};
+
+const useFakeTime = () => {
+  const currentDateTime = new Date();
+  currentDateTime.setHours(11, 30);
+
+  jest.useFakeTimers();
+  jest.setSystemTime(currentDateTime);
+};
+useFakeTime();
+
+describe("badge for the previous day", () => {
```

Comment:
i guees we should wrap your 3 describe functions with describe function named `"GIVEN TimeBadge"`
@okhmelevskyi  can we do as I wrote?